project/make/gccgo

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -e
+
+DEST=$1
+BINARY_NAME="docker-$VERSION"
+BINARY_EXTENSION="$(binary_extension)"
+BINARY_FULLNAME="$BINARY_NAME$BINARY_EXTENSION"
+
+source "$(dirname "$BASH_SOURCE")/.go-autogen"
+
+if [ "$IAMSTATIC" == "true" ]; then 
+	EXTLDFLAGS_STATIC_DOCKER="-static -Wl,--no-export-dynamic $(echo $EXTLDFLAGS_STATIC_DOCKER | sed "s/$EXTLDFLAGS_STATIC//")"
```

Comment:
Same comment as above.. EXTLDFLAGS_STATIC_DOCKER is already set in make.sh. Do we really need to reset it again here ?